sdk/go/auto/local_workspace.go

```diff
@@ -608,6 +608,21 @@ func (l *LocalWorkspace) ListStacks(ctx context.Context) ([]StackSummary, error)
 	return stacks, nil
 }
 
+// ListAllStacks returns all Stacks the current logged-in Pulumi identity has access to.
+// This queries underlying backend and may return stacks not present in the Workspace (as Pulumi.<stack>.yaml files).
+func (l *LocalWorkspace) ListAllStacks(ctx context.Context) ([]StackSummary, error) {
+	var stacks []StackSummary
+	stdout, stderr, errCode, err := l.runPulumiCmdSync(ctx, "stack", "ls", "--json", "--all")
+	if err != nil {
+		return stacks, newAutoError(fmt.Errorf("could not list all stacks: %w", err), stdout, stderr, errCode)
+	}
+	err = json.Unmarshal([]byte(stdout), &stacks)
+	if err != nil {
+		return nil, fmt.Errorf("unable to unmarshal all stacks value: %w", err)
+	}
+	return stacks, nil
+}
```

Comment:
Nit: Consider adding an unexported helper function, e.g. `func (l *LocalWorkspace) listStacks(ctx context.Context, all bool) ([]StackSummary, error)`, to share most of the code between these two.